lib/alephant/lookup/lookup_cache.rb

```diff
@@ -29,7 +29,8 @@ def get(key, &block)
           logger.info "Alephant::LookupCache#get key: #{versioned_key} - #{result ? 'hit' : 'miss'}"
           logger.metric "GetKeyMiss" unless result
           result ? result : set(key, block.call)
-        rescue StandardError => e
```

Comment:
I wonder if this should be more specific. It seems the error would be if the item is not in the cache so would call the block to fetch the item from source.

So this will log when an item is not in the cache?